lightning-invoice/src/payment.rs

```diff
@@ -731,15 +731,15 @@ where
 mod tests {
 	use super::*;
 	use crate::{InvoiceBuilder, Currency};
-	use crate::utils::{ScorerAccountingForInFlightHtlcs, create_invoice_from_channelmanager_and_duration_since_epoch};
+	use crate::utils::create_invoice_from_channelmanager_and_duration_since_epoch;
 	use bitcoin_hashes::sha256::Hash as Sha256;
 	use lightning::ln::PaymentPreimage;
 	use lightning::ln::channelmanager;
 	use lightning::ln::features::{ChannelFeatures, NodeFeatures};
 	use lightning::ln::functional_test_utils::*;
 	use lightning::ln::msgs::{ChannelMessageHandler, ErrorAction, LightningError};
 	use lightning::routing::gossip::{EffectiveCapacity, NodeId};
-	use lightning::routing::router::{InFlightHtlcs, PaymentParameters, Route, RouteHop, Router};
+	use lightning::routing::router::{InFlightHtlcs, PaymentParameters, Route, RouteHop, Router, ScorerAccountingForInFlightHtlcs};
```

Comment:
It's a little strange that we test `ScorerAccountingForInFlightHtlcs` in this crate still in `considers_inflight_htlcs_between_retries`. Probably should move the test.

Reply:
Could we save that since all of the tests are going to be moving soon?